### ISSUE TITLE
[DataCap Application] Smithsonian Open Access

### ISSUE THREAD

**Issue by Megan008** [filecoin-project/filecoin-plus-large-datasets#1362]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: Public Data-Smithsonian Open Access
- Website / Social Media:https://registry.opendata.aws/smithsonian-open-access/
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB):200TiB
- On-chain address for first allocation:f15ghjf5pyqnsfgc2rdovno5jwjofoebsqdidi5bq

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
I have participated in some projects and hackathon. I have experience on it.
```

What is the primary source of funding for this project?
```
Personal income.
```

What other projects/ecosystem stakeholders is this project associated with?
```
No.
```

## Use-case details

Describe the data being stored onto Filecoin
```
The edan content repository [edanCR] is a centrally administered data store comprised of millions of records contributed from a wide variety of Smithsonian sources including collection management systems, web applications, internal databases and externally authored content.
```

Where was the data in this dataset sourced from?
```
Smithsonian sources https://edan.si.edu/openaccess/docs/
https://registry.opendata.aws/smithsonian-open-access/
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
s3://smithsonian-open-access/3d/002f2567-7384-4027-9b99-bcb7a7e6361e/USNM153798_cranium_-400-150k-1024-normals.jpg
s3://smithsonian-open-access/3d/00416a1c-358a-4f5b-9a90-053df0d6fd31/USNM142171_Mandible_-300-150k-4096-normals.jpg
s3://smithsonian-open-access/3d/009463d3-6f58-4f5b-8e60-915805a876ee/USNM_91201-150k-2048-medium.glb
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```

Yes, it is the AWS open dataset.
```

What is the expected retrieval frequency for this data?
```
Not very often.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
2 years.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
North america; Korea; China.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
75% data will be distributed by offline data transfer. Other data will use online transfer for distributing with storage providers who close to me.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
I would let 1 sp who used to cooperate with me for this deal. Now I'm chatting with other sps.
```

How will you be distributing deals across storage providers?
```
I have communicated with 4 sp. In first time, I will divide 1/4 data to each sp. If I find out more sp, I will decrease the percentage of deals to them --- for decentralized storage.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes.
```

**Comment by simonkim0515** [comment]
### Datacap Request Trigger
**Total DataCap requested**

> 5PiB

**Expected weekly DataCap usage rate**

> 200TiB

**Client address**

>f15ghjf5pyqnsfgc2rdovno5jwjofoebsqdidi5bq

**Comment by cryptowhizzard** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecfa74u24dm3nfp2ts3j4os2osmclwkckhkmajyjewho4axjvgnnm 
#### Address 
> f15ghjf5pyqnsfgc2rdovno5jwjofoebsqdidi5bq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
#### Id
> 9b8eeb35-4daf-41f3-910f-fec518517047
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecfa74u24dm3nfp2ts3j4os2osmclwkckhkmajyjewho4axjvgnnm

**Comment by kernelogic** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb2eqbl2p76uck37almav2tmvztlwagzgyq3eort5lmyxsxjz2mni 
#### Address 
> f15ghjf5pyqnsfgc2rdovno5jwjofoebsqdidi5bq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> 9b8eeb35-4daf-41f3-910f-fec518517047
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb2eqbl2p76uck37almav2tmvztlwagzgyq3eort5lmyxsxjz2mni

**Comment by BDEio** [comment]
@Megan008  Hi! Congratulations on your DataCap approval！
BDE is a trusted deals auction house helping you to get paid storing your data with reliable storage providers. If you need any help, please get in touch.

**Comment by cryptowhizzard** [comment]
Hello @Megan008 

I approved the first allocation of this LDN to see where it was going as the dataset you were / are storing is a great dataset and valuable for humanity.

I have checked in again on this LDN and notice that it is not stored in line with FIL+ rules.
- The content is not retrievable.
- The distribution is not across multiple providers in different regions, hence everything has been stored with one SP.

Therefore i can't support this LDN application anymore and won't sign on any subsequent requests.

Thank you for your understanding.

[f01984693.csv.log](https://github.com/filecoin-project/filecoin-plus-large-datasets/files/10421104/f01984693.csv.log)

**Comment by Meibuy** [comment]
checker:manualTrigge

**Comment by MetaWaveInfo** [comment]
This check-bot result is not accurate, and the retrieval can be carried out. Hope to see more regional nodes in the next round.
![image](https://user-images.githubusercontent.com/100760052/213631800-d31b79d5-e214-46ef-a9e6-733aa8cd396c.png)

**Comment by MetaWaveInfo** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedavw54tt2hj6qr6vulujddhiqqol2olp2irlilmgmhi4e22netee 
#### Address 
> f15ghjf5pyqnsfgc2rdovno5jwjofoebsqdidi5bq
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1ktlkcxnmzxcdaoqfsunrg3vocfbmgv4n3mrn74a
#### Id
> 812c09bc-9678-4352-804c-dfff5ded3033
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedavw54tt2hj6qr6vulujddhiqqol2olp2irlilmgmhi4e22netee

**Comment by cryptowhizzard** [comment]
Hello

All above miners are in the same IP address We cannot proceed here as this set is only stored at one SP / organization.

@MetaWaveInfo please cancel your proposal asap.

Second .... Fil+ states readily retrieval. I am already waiting on hour to get something retrieved but no-avail. It means that fast retrieval is disabled here. This has not been reported in the LDN application itself.



lotus net connect f01939377
f01939377 -> {12D3KooWP6ruBqoH971N6goZRdaHnSyRyeTxejH7nzjKktVJqjgf: [/ip4/183.220.36.2/tcp/36615]}
ERROR: failed to parse multiaddr "f01939377": must begin with /

lotus net connect f01939387
f01939387 -> {12D3KooWCsXpYdxt3HB8xUgBb6Pa9Xot9xnd8v7URzcJRVLDy3Rc: [/ip4/183.220.36.2/tcp/36616]}
ERROR: failed to parse multiaddr "f01939387": must begin with /

lotus net connect f01941622                                                                                                                                                                                                                                                                                                       
f01941622 -> {12D3KooWSXrsMKNfCsTsGydFj1Ha8KmRqg1q2M3bmFmeDFAoyWuE: [/ip4/183.220.36.2/tcp/36617]}

>>>>>

cat /root/Dropbox/clientdealscannerdone/f01984693.csv.log                                                                                                                                                                                                                                                                         |
/usr/local/bin/lotus state get-deal 21964415/usr/local/bin/lotus client retrieve --provider f01757676 mAXCg5AIgVFyhJC3sVa71XuGh/+KbyrCkULkNZJ/G1MENj2FUrCs
/usr/local/bin/lotus state get-deal 21964416/usr/local/bin/lotus client retrieve --provider f01757676 mAXCg5AIgS+VTiTfMCXmrBzluCdZZwgFAoYdgrIACyjph7zT+C4A
/usr/local/bin/lotus state get-deal 21967414/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIga7GWy40SW2cm3adszZoj0Za470g304mJJ/nqFvqeR+g
/usr/local/bin/lotus state get-deal 21967416/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIgGZfV6VbPf10sWRLaAAOka8ci2+DyjecPxD0PiUoCh1U
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556086|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 9ms [1673908265200556086|0]
/usr/local/bin/lotus state get-deal 21981355/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIgKtYw+A76Ia/6yueuIgBEsgY1xedhGKpujfilRElZsFk
Recv 0 B, Paid 0 FIL, DealAccepted (Accepted), 373ms [1673908265200556086|0]
Recv 0 B, Paid 0 FIL, PaymentChannelSkip (Ongoing), 373ms [1673908265200556086|0]
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556087|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 9ms [1673908265200556087|0]
/usr/local/bin/lotus state get-deal 21981358/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIgg4zIAEY5qIJg5R3yB6vYNX3hCzJocyc9vS0nJNbeFVA
Recv 0 B, Paid 0 FIL, DealAccepted (Accepted), 364ms [1673908265200556087|0]
Recv 0 B, Paid 0 FIL, PaymentChannelSkip (Ongoing), 364ms [1673908265200556087|0]
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556088|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 9ms [1673908265200556088|0]
/usr/local/bin/lotus state get-deal 21992957/usr/local/bin/lotus client retrieve --provider f01941622 mAXCg5AIgmOVyBHce3VEkaR69tg+4V5of9nGOYn200gh1wYiUFj8
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556089|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 9ms [1673908265200556089|0]
/usr/local/bin/lotus state get-deal 21992958/usr/local/bin/lotus client retrieve --provider f01941622 mAXCg5AIguhrAxRgb3UlY7DDgUotF9uG2yjTqkLpzvH0LJsTX520
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556090|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 12ms [1673908265200556090|0]
/usr/local/bin/lotus state get-deal 22001706/usr/local/bin/lotus client retrieve --provider f01941622 mAXCg5AIg4GOJ2kZ1S6uFxN4voJAj2kcP4SMXPSd+js2ST7r867w
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556091|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556091|0]
/usr/local/bin/lotus state get-deal 22001707/usr/local/bin/lotus client retrieve --provider f01941622 mAXCg5AIgzlvR4QlbjKzseIKgM3UAKpiwYP41BAS/f9Okefbao6g
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556092|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 13ms [1673908265200556092|0]
/usr/local/bin/lotus state get-deal 22010721/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIgJbn7cCww/1mDPRYUrjUareU36Yrr7PwuZjc6DQasju4
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556093|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556093|0]
/usr/local/bin/lotus state get-deal 22010722/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIgCQliEvE/UEh1tYaHY7CykAxYcTXnGGhQpqt8mc9tY14
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556094|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556094|0]
/usr/local/bin/lotus state get-deal 22020323/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIg45yzrLaST8YyULQbtH831c3veNJ8WjLyomnp922SshU
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556095|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556095|0]
/usr/local/bin/lotus state get-deal 22020324/usr/local/bin/lotus client retrieve --provider f01939387 mAXCg5AIg6Wqx51p1x73keh3o9J30/JBuJerwe/N313VThnSDEKw
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556096|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 9ms [1673908265200556096|0]
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556097|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556097|0]
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556098|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556098|0]
Recv 0 B, Paid 0 FIL, Open (New), 0s [1673908265200556099|0]
Recv 0 B, Paid 0 FIL, DealProposed (WaitForAcceptance), 10ms [1673908265200556099|0]

**Comment by cryptowhizzard** [comment]
@Megan008 

I am sorry for the technical language in my post above for a fellow notary who did not do proper duediligence.

For the Filecoin + program itself please note that by storing data on the Filecoin network you get 10x the power ( QAP ) of a normal sector. In return for this to be eligible for this incentive there are important rules to abide:

The data must be stored with multiple service providers. ( Not with one organization running multiple SP's but really independent organizations )
The data must be distributed over multiple regions. (Running a VPN is not allowed to fake presence in the USA for example)
The data must be retrievable by everyone in the community to check if you are really storing what you say you are storing. Running boost is highly recommended.
The data must be in line with the mission of FIL+. That means that the data must be valuable for humanity. It can be scientific or other things that people are highly interested in and is unique.
As data preparer you are allowed to store one copy yourself (if the client requests so for the sake of redundancy of the data). If you store your copy in China it means that the other organizations/SP must be outside China.

Above shows that the rules have not been followed in this LDN. Let me know if you have questions about that. 

If not then i would like to create a data onboarding plan together with you to recover and get this LDN on the right track. 
It consists of:

The entity that is going to do the data preparation for you, where and when. If you need recommendations i can give them to you.
The new SP's you are going to store data with and their reputation. If you need recommendations i can give them to you.
When we do checks on the data stored and how and by who.
The retrieval frequency (how often does the data needs to be retrieved and from where)

**Comment by Megan008** [comment]
@cryptowhizzard Thank you for reminding us that we are doing this. The quota for the first round is very small, and it is estimated that it will take the fourth round to meet this requirement. We welcome more notaries who are willing to accept this plan to sign.

**Comment by cryptowhizzard** [comment]
Hi @Megan008 

I don't think we speak the same the same language.

Can you please enlighten us with the new SP's you will use for next round of allocation? We cannot allocated with the existing SP's in place.

**Comment by Megan008** [comment]
@cryptowhizzard Not yet, but you can see it in the next round of check report. We are communicating with several SPs, but the premise of cooperation is having datacap.

**Comment by cryptowhizzard** [comment]
@Megan008 

The thing is that this ID's ( i believe those belong to you or a single organisation ) are heavily involved in CID sharing as you can see here -> https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/888
The same set of SP's is also used in #650 and #653 who say to be partners and store the same unique data, however their applications stated that they were going to store something else.

All with all i think the principles of limited thrust should apply here. You got the thrust from me a while back. You did not adhere to the FIL+ rules. We cannot continue on the same bases as where we were.

Again:

Above shows that the rules have not been followed in this LDN. Let me know if you have questions about that.
If not then i would like to create a data onboarding plan together with you to recover and get this LDN on the right track.

It consists of:
The entity that is going to do the data preparation for you, where and when. If you need recommendations i can give them to you.
The new SP's you are going to store data with and their reputation. If you need recommendations i can give them to you.
When we do checks on the data stored and how and by who.
The retrieval frequency (how often does the data needs to be retrieved and from where)

"The next round" is not acceptable.

**Comment by Megan008** [comment]
@cryptowhizzard We don't own any nodes. Please don't make malicious guesses and randomly interfere with other notaries to sign. We will distribute them in accordance with the compliance way, and welcome to check the result after the third or forth round.

**Comment by cryptowhizzard** [comment]
Ok, i have waited over the weekend to give you time to come clean.

https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1013

![Scherm­afbeelding 2023-01-23 om 11 35 29](https://user-images.githubusercontent.com/36404435/214018845-c3261cd3-667d-4fba-8263-0d67c2043db7.png)

There is no reason to give you further trust since your applications have also been heavily involved in CID sharing. Enough is enough.

I will start writing a dispute against @MetaWaveInfo today.

**Comment by kernelogic** [comment]
We should have a shared client blacklist for things like this. CID sharing is absurd. Don't tell me they all stole from your datasets.

**Comment by Megan008** [comment]
@kernelogic CID sharing is a way for some SPs to save costs, which is really inappropriate. We have asked for rectification with the SPs that cooperates with us.  By the way, I saw that your applications also had a large number of CIDs shared with different data subjects. Please do not be critical with other applicants if you can't be perfect.

**Comment by kernelogic** [comment]
Sharing between different business entities and sharing between your own are two different scenarios.

**Comment by MetaWaveInfo** [comment]
First, I verified that the data was retrievable before I signed it. Second, this is the first round of allocation. As the applicant promised will reach decentralized storage at the forth round,  I don't think it helps to label it prematurely. Thanks for the heads up, but it's quite common that first round isn't decentralized sufficiently. You can open a new issue for discussion if you think the allocation requirment needs to be improved. It will be helpful to clarify the community rules and reduce confusion for all notaries.
As an open community, each notary has their own criteria. I hope more notaries can validate the application in later allocations. We will continue to follow up on the issue and provide guidance based on the subsequent allocation.

**Comment by Megan008** [comment]
@MetaWaveInfo thanks for your trust. I will follow FIL+ rule as i promised in KYC.
As far as I understand, there is no community requirement to have a complete list of SPs in the process of application (please correct me if I'm wrong). The first round of allocation is  just finished and I am still communicating with many of the SPs as mentioned in the application and looking for more through Filmine, including @BDEio above, etc. I will prioritize the SPs that can join our application to meet the multi-region allocation requirement as expected.
@herrehesse yes, my previous application #1013 involved heavy cid sharing. The SPs I worked with did not follow the distribution as communicated and there've been no explanation offered. I recognize that their behavior was harmful to the ecosystem and I am not considering working with them again in the future. I closed the application just now and I will be more stringent in selecting SPs!

**Comment by MetaWaveInfo** [comment]
Thank you for your understanding and cooperation. I believe that everyone is willing to develop community for the better, and everyone will benefit from it!

**Comment by dongpo313** [comment]
checker:manualTrigger

**Comment by Megan008** [comment]
keep open.

**Comment by Sunyaoyaomk** [comment]
checker:manualTrigger
